src/features/import/components/Importer/validation/importAlert.tsx

```diff
@@ -0,0 +1,87 @@
+import { Circle } from '@mui/icons-material';
+import {
+  Alert,
+  AlertTitle,
+  Box,
+  Button,
+  Checkbox,
+  FormControlLabel,
+  Typography,
+} from '@mui/material';
+
+import messageIds from 'features/import/l10n/messageIds';
+import { useMessages } from 'core/i18n';
+
+export enum ALERT_STATUS {
+  ERROR = 'error',
+  WARNING = 'warning',
+  INFO = 'info',
+  SUCCESS = 'success',
+}
+interface ImportAlertProps {
+  bullets?: string[];
+  msg: string;
+  status: ALERT_STATUS;
+  title: string;
+  onClickBack?: () => void;
+  onChecked?: () => void;
+}
+const ImportAlert: React.FunctionComponent<ImportAlertProps> = ({
+  bullets,
+  msg,
+  status,
+  title,
+  onChecked,
+  onClickBack,
+}) => {
+  const message = useMessages(messageIds);
+
+  return (
+    <Alert
+      action={
+        status === ALERT_STATUS.ERROR && (
+          <Button
+            color="inherit"
+            onClick={onClickBack}
+            size="small"
+            sx={{ fontWeight: 'bold', whiteSpace: 'nowrap' }}
+            variant="text"
+          >
+            {message.validation.alerts.back()}
+          </Button>
+        )
+      }
+      severity={status}
+    >
+      <AlertTitle>
+        <strong>{title}</strong>
```

Comment:
I think  you could style this title using the `sx` of this component, or put the title text in a `Typography` that you style?